commit b3cf94ef15f0bb6ff336907e31ab0064a0381916
Author: Stafford Horne <shorne@gmail.com>
Date:   Wed Oct 13 21:43:50 2021 +0900

    login: Add back libutil as an empty library
    
    There are several packages like sysvinit and buildroot that expect
    -lutil to work.  Rather than impacting them with having to change
    the linker flags provide an empty libutil.a.

diff --git a/login/Makefile b/login/Makefile
index 4e6b97734d..e2bdcb1b0d 100644
--- a/login/Makefile
+++ b/login/Makefile
@@ -46,10 +46,10 @@ vpath %.c programs
 tests := tst-utmp tst-utmpx tst-grantpt tst-ptsname tst-getlogin tst-updwtmpx \
   tst-pututxline-lockfail tst-pututxline-cache
 
-ifeq ($(have-GLIBC_2.33),yes)
 # Empty compatibility library for old binaries.
 extra-libs      := libutil
 extra-libs-others := $(extra-libs)
+ifeq ($(have-GLIBC_2.33),yes)
 libutil-routines := libutil-compat
 libutil-shared-only-routines := libutil-compat
 
@@ -57,6 +57,8 @@ libutil-shared-only-routines := libutil-compat
 # link is not installed.
 install-lib-ldscripts = libutil.so
 $(inst_libdir)/libutil.so:
+else  # not $(have-GLIBC_2.33)
+libutil-inhibit-o = $(filter-out .o,$(object-suffixes))
 endif # $(have-GLIBC_2.33)
 
 include ../Rules
